hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OutboundResponseHandler.java

```diff
@@ -87,15 +119,123 @@ public boolean send(Response response, Address target) {
 
         byte[] bytes = serializationService.toBytes(response);
         Packet packet = new Packet(bytes, -1)
-                .setPacketType(Packet.Type.OPERATION)
+                .setPacketType(OPERATION)
                 .raiseFlags(FLAG_OP_RESPONSE);
 
         if (response.isUrgent()) {
             packet.raiseFlags(FLAG_URGENT);
         }
 
+        return transmit(target, packet);
+    }
+
+    private boolean sendNormalResponse(Address target, long callId, int backupAcks, boolean urgent, Object value) {
+        checkTarget(target);
+
+        Packet packet = toNormalResponsePacket(callId, (byte) backupAcks, urgent, value);
+
+        return transmit(target, packet);
+    }
+
+    Packet toNormalResponsePacket(long callId, int backupAcks, boolean urgent, Object value) {
+        byte[] bytes;
```

Comment:
Ok, good that we have a test meaning that it will fail if someone tries to change the serialization format and forgets to change it in all places. Looking forward to moving the responses away from IDS and more internally.